bower.json

```diff
@@ -17,10 +14,9 @@
     "output",
     "test",
     "bower.json",
-    "gulpfile.js",
     "package.json"
   ],
   "dependencies": {
-    "purescript-prelude": "^0.1.0"
+    "purescript-prelude": "^1.0.0"
```

Comment:
Definitely :+1: I just wanted to get the version in these PRs initially right at least.

Ideally I'd like to proceed with these by: first releasing 0.8[*], so travis can build these, and then checking each one with travis before merge/release, so that means proceeding in topological order too. I've got a whole bunch of stuff setup locally which definitely builds together, but I'd like to double check with things actually being fetchable with bower to ensure the versions are all right, etc.

[*] whenever we do do that, I'm not saying we rush into it right now :smile: - just thought I'd open PRs to help keep track of which things I've already worked on, and also to check if there are issues we want to fold in to fix as part of the 1.0.0 update